src/client/assets/icons/icons.js

```diff
@@ -28,3 +29,4 @@ export const stopIcon = <FontAwesomeIcon icon={faStop} className="icon" />;
 export const playIcon = <FontAwesomeIcon icon={faPlay} className="icon" />;
 export const pauseIcon = <FontAwesomeIcon icon={faPause} className="icon" />;
 export const cancelWindowIcon = <FontAwesomeIcon icon={faTimes} />;
+export const checkTick = <FontAwesomeIcon icon={faCheck} />;
```

Comment:
use a classic type of naming icons please, like checkTickIcon)

Reply:
fixed